core/game/treepath_test.go

```diff
@@ -1,14 +1,15 @@
-package treepath
+package game_test
```

Comment:
Shouldn't this be package game?

Reply:
It's intentionally not. This depends on SGF (for convenience), and SGF depends on game. Using game_test (separate package for tests), fixes the cycle.